docs/modules/ROOT/pages/governance.adoc

```diff
@@ -102,7 +102,7 @@ A proposal is a sequence of actions that the Governor contract will perform if i
 
 Let’s say we want to create a proposal to give a team a grant, in the form of ERC-20 tokens from the governance treasury. This proposal will consist of a single action where the target is the ERC-20 token, calldata is the encoded function call `transfer(<team wallet>, <grant amount>)`, and with 0 ETH attached.
 
-Generally a proposal will be created with the help of an interface such as Tally or Defender. Here we will show how to create the proposal using Ethers.js.
+Generally a proposal will be created with the help of an interface such as Tally or https://defender.openzeppelin.com/v2/#/actions/executable[Defender]. Here we will show how to create the proposal using Ethers.js.
```

Comment:
Same rationale as before. Also specifying "Defender Proposals" for clarity, otherwise the link should go to the Defender landing page imo

```suggestion
Generally a proposal will be created with the help of an interface such as Tally or https://docs.openzeppelin.com/defender/module/actions#transaction-proposals-reference[Defender Proposals]. Here we will show how to create the proposal using Ethers.js.
```